pom.xml

```
@@ -1345,6 +1345,9 @@
                             </argLine>
                             <useManifestOnlyJar>false</useManifestOnlyJar>
                             <useSystemClassLoader>true</useSystemClassLoader>
+                            <includes>
+                                <include>**/**IT.java</include>
```

Any reason to use the explicit definition?
The default contains the expression too:
```xml
<includes>
    <include>**/IT*.java</include>
    <include>**/*IT.java</include>
    <include>**/*ITCase.java</include>
</includes>
```

---

One more thing - the `**` in the filename is unnecessary.

---

Yes, to clearly differentiate between surefire and failsafe. During investigation I had very weird behaviours where some tests were not executed at all and some were executed twice; making config explicit made all tests pass exactly once.
And yes, I used only IT suffix as it's the most commonly used. Should I add the rest of them?

---

OK, it make sense to keep it simple - so no additional expressions. I would just use the single star in the file name:
```suggestion
                                <include>**/*IT.java</include>
```